src/test/scala/no/ndla/conceptapi/controller/DraftConceptControllerTest.scala

```diff
@@ -116,13 +116,13 @@ class DraftConceptControllerTest extends UnitSuite with TestEnvironment with Sca
       .thenReturn(Success(TestData.sampleNbApiConcept))
 
     val missing = """{"language":"nb"}"""
-    val missingExpected = TestData.emptyApiUpdatedConcept.copy(language = "nb", articleId = Right(None))
+    val missingExpected = TestData.emptyApiUpdatedConcept.copy(language = "nb", articleIds = None)
 
-    val nullArtId = """{"language":"nb","articleId":null}"""
-    val nullExpected = TestData.emptyApiUpdatedConcept.copy(language = "nb", articleId = Left(null))
+    val nullArtId = """{"language":"nb","articleIds":null}"""
+    val missingExpectedWhenNull = TestData.emptyApiUpdatedConcept.copy(language = "nb", articleIds = null)
 
-    val existingArtId = """{"language":"nb","articleId":10}"""
-    val existingExpected = TestData.emptyApiUpdatedConcept.copy(language = "nb", articleId = Right(Some(10)))
+    val existingArtId = """{"language":"nb","articleIds":[10]}"""
+    val existingExpected = TestData.emptyApiUpdatedConcept.copy(language = "nb", articleIds = Some(Seq(10)))
```

Comment:
Denne testen må nok skrives om til å teste på et annet felt som bruker den `Deletable` typen for å fortsatt teste det samme, feks `metaImage` feltet (Dvs reverte testen til sånn den så ut tidligere og bare bytte ut `articleId` med `metaImage`.